assets/src/js/frontend/paypal-commerce/SmartButtons.js

```diff
@@ -36,6 +37,7 @@ class SmartButtons extends PaymentMethod {
 				color: 'gold',
 				tagline: false,
 			},
+			funding: { disallowed: [ paypal.FUNDING.CARD, paypal.FUNDING.CREDIT ] },
```

Comment:
We know that this doesn't work. It's also not really in scope with this PR. Let's remove this (and the comment above), for now.

Reply:
fixed: https://github.com/impress-org/givewp/compare/896c0482ddb9...30b522638fc9